sbol2/object.py

```diff
@@ -469,6 +469,12 @@ def __getattribute__(self, name):
             # Convert the ReferencedObject to a value instead of
             # returning the ReferencedObject itself
             result = result.value
+        elif isinstance(result, OwnedObject):
+            sbol_property = object.__getattribute__(self, name)
+            if sbol_property.getUpperBound() == '1':
```

Comment:
Having bounds as strings is really not useful. Properties also have `upper_bound` and `lower_bound` attributes that convert the bounds to ints for easier comparison. You could do this:
```suggestion
            if sbol_property.upper_bound == 1:
```
I think that's a lot clearer from an intention standpoint.